x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/utils/get_finding_colors.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiThemeComputed } from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme'; // TODO: replace with euiTheme
+import type { VulnSeverity } from '@kbn/cloud-security-posture-common';
+import {
+  VULNERABILITIES_SEVERITY,
+  MISCONFIGURATION_STATUS,
+} from '@kbn/cloud-security-posture-common';
+
+const isBorealis = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('borealis');
+};
+
+const isAmsterdam = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('amsterdam');
+};
+
+// TODO: replace with severity color palette
+// TODO: replace with euiTheme.colors.vis.* from useEuiTheme hook
+// TODO: replace #aaa with proper color token
+export const getSeverityStatusColor = (
+  severity: VulnSeverity,
+  euiTheme: EuiThemeComputed
+): string => {
+  // TOOD: remove old mapping when severity palette is fixed (atm it is inverted)
```

Comment:
we should be able to just use the new severity color palette for both Amsterdam and Borealis, but currently there is a bug in https://github.com/elastic/eui/pull/8247 that the severity colors for Amsterdam are inverted. Keeping the old mapping until the severity color palette is fixed